x-pack/platform/plugins/shared/integration_assistant/server/integration_builder/readme_files.ts

```diff
@@ -58,12 +59,30 @@ function createReadmeFile(
   });
 
   const template = env.getTemplate(templateName);
+  const uniqueInputs = getUniqueInputs(datastreams);
 
   const renderedTemplate = template.render({
     package_name: integrationName,
     datastreams,
     fields,
+    inputs: uniqueInputs,
+    include_ssl: shouldIncludeSSLDocumentation(uniqueInputs),
   });
 
   createSync(joinPath(targetDir, 'README.md'), renderedTemplate);
 }
+
+function getUniqueInputs(datastreams: DataStream[]): string[] {
+  const uniqueInputs = new Set<string>();
+  datastreams.forEach((datastream) => {
+    datastream.inputTypes.forEach((inputType) => {
+      uniqueInputs.add(inputType);
+    });
+  });
+
+  return Array.from(uniqueInputs);
```

Comment:
This can be much written much more expressively in TypeScript, e.g. Sonnet suggests 

```ts
return [...new Set(datastreams.flatMap(d => d.inputTypes))];
```